test/test_examples_2d_old.jl

```diff
@@ -175,16 +175,16 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
             l2   = [0.05365734539276933, 0.04683903386565478, 0.04684207891980008, 0.19632055541821553],
             linf = [0.18542234326379825, 0.24074440953554058, 0.23261143887822433, 0.687464986948263])
   end
-  @testset "taal-check-me parameters_euler_khi_shockcapturing.toml" begin
+  @testset "taal-confirmed differences-to-master-accepted parameters_euler_khi_shockcapturing.toml" begin
```

Comment:
Thanks for specifically confirming that the differences to master are all accounted for. In this case, I think it's ok to remove the `differences-to-master-accepted`, since from now on we will not mark anything as `confirmed` otherwise.